packages/sirius-web/backend/sirius-web/src/test/resources/scripts/initializeMigrationParticipant.sql

```diff
@@ -0,0 +1,52 @@
+-- Sample studio project
+INSERT INTO project (
+  id,
+  name,
+  created_on,
+  last_modified_on
+) VALUES (
+  '01234836-0902-418a-900a-4c0afd20323e',
+  'Studio',
+  '2024-01-01 9:42:0.000',
+  '2024-01-02 9:42:0.000'
+);
+INSERT INTO nature (
+  project_id,
+  name
+) VALUES (
+  '01234836-0902-418a-900a-4c0afd20323e',
+  'siriusComponents://nature?kind=studio'
+);
+INSERT INTO semantic_data (
+  id,
+  project_id,
+  created_on,
+  last_modified_on
+) VALUES (
+  'e344d967-a639-4f6c-9c00-a466d51063c6',
+  '01234836-0902-418a-900a-4c0afd20323e',
+  '2024-01-01 9:42:0.000',
+  '2024-01-02 9:42:0.000'
+);
+INSERT INTO semantic_data_domain (
+  semantic_data_id,
+  uri
+) VALUES (
+  'e344d967-a639-4f6c-9c00-a466d51063c6',
+  'http://www.eclipse.org/sirius-web/view'
+);
```

Comment:
Is that correct? It seems that you are also using the diagram part in the document